### ISSUE TITLE
Add attestation as a property under security-artifacts

### ISSUE THREAD

**Issue by Danajoyluck** [ossf/security-insights-spec#83]
For software attestation to be added to SECURITY_INSIGHTS.yml, it seems  it needs to go under “security-artifacts” > “other-artifacts”. It would be nice to have attestation as an explicit property, similar as SBOM.  It will make it a deterministic property for policy engines to pick it up and make decisions during software ingestion.

**Comment by eddie-knight** [comment]
I've drafted this into #96:

```yaml
project:
  release:
    latest:
      sbom:
        - url: https://foo.bar/sbom1
          format: cyclonedx
        - url: https://foo.bar/sbom2
          format: spdx
```

**Comment by eddie-knight** [comment]
Thanks for this suggestion @Danajoyluck! This was added in the [attestation object](https://github.com/ossf/security-insights-spec/blob/main/specification-details/aliases.md#attestation) which will be included in the upcoming release (#97).
